analytical/tests/test_tag_chartbeat.py

```diff
@@ -4,6 +4,7 @@
 
 import re
 
+from django import version
```

Comment:
This line causes the issues. If you use try-except below a couple of failing builds will vanish.